src/search-modal/SearchResult.jsx

```diff
@@ -149,10 +144,16 @@ const SearchResult = ({ hit }) => {
       return `/${urlSuffix}`;
     }
 
-    if (usageKey.startsWith('lb:')) {
-      if (redirectToLibraryAuthoringMfe) {
-        return getLibraryHitUrl(hit, libraryAuthoringMfeUrl);
+    if (contextKey.startsWith('lib:')) {
+      const urlSuffix = getLibraryComponentUrlSuffix(hit);
+      if (libraryAuthoringMfeUrl) {
+        return `${libraryAuthoringMfeUrl}${urlSuffix}`;
       }
```

Comment:
When clicking on the search result it still redirects to the library authoring mfe when the flag is disabled. I think it missing `&& redirectToLibraryAuthoringMfe` in the check.

Reply:
Fixed: c2bdecfae35bea464a439f3043faddd305c76f01